mico-core/src/main/java/io/github/ust/mico/core/WebConfig.java

```diff
@@ -26,8 +31,16 @@
     public FilterRegistrationBean<CorsFilter> corsFilter() {
         UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
         CorsConfiguration config = new CorsConfiguration().applyPermitDefaultValues();
+        config.addAllowedMethod(HttpMethod.DELETE);
+        config.addAllowedMethod(HttpMethod.PUT);
+        for (String additionalAllowedMethod : corsConfig.getAdditionalAllowedMethods()) {
+            log.info("Adding additional method to CORS config:"+ additionalAllowedMethod);
+            config.addAllowedMethod(additionalAllowedMethod);
```

Comment:
what exactly is this loop doing? Seams as if it would add values into the list they came from. Please correct me if I'm wrong...